packages/telegraf/extra/dcos-telegraf.service

```diff
@@ -19,6 +19,8 @@ LimitNOFILE=16384
 ExecStartPre=/bin/bash -c "mkdir -p /run/dcos/telegraf"
 ExecStartPre=/bin/bash -c "chmod 775 /run/dcos/telegraf"
 ExecStartPre=/bin/bash -c "chown root:dcos_telegraf /run/dcos/telegraf"
+# The metrics isolator module dir may not exist; if so, chown it to dcos_telegraf; if not, no-op.
+ExecStartPre=/bin/bash -c "chown -R dcos_telegraf /run/dcos/mesos/isolators/com_mesosphere_MetricsIsolatorModule || :"
```

Comment:
Can we get away with doing the migration itself as root in this step? That seems less risky than changing the ownership of a runtime directory that belongs to Mesos.

Reply:
Yes, that should work. I feel like there is a reason we didn't do it that way to begin with, but it's lost in the mists of time and the original PR doesn't help.

Reply:
Yeah I don't recall either. Could just be that the startup script was an easy place to put a bit of bash.

Reply:
Entirely possible :grimacing: 

I've made the change you suggested.